### PR TITLE
Replace Toast with SnackBar in Denied Permissions

### PR DESCRIPTION
**Pull request by ahmedadeltito**
Replace toast with snack bar in case of denied permissions.

### Changes
- Create Snack Bar in `Util` Class.
- Add new strings in the `strings.xml` file. We will need to translate it.

### Reviewers
@e2po @ConorOD @patrick-doyle @ReguloSar

### References
- None

### Risks
- None

### CODE REVIEW

File: belvedere/src/main/java/zendesk/belvedere/BelvedereUi.java [hunk 1]
```diff
@@ -210,9 +212,29 @@ public void run() {
...
+                                Snackbar.LENGTH_LONG,
+                                appCompatActivity.getString(R.string.belvedere_navigate_to_settings),
+                                new OnClickListener() {
+                                    @Override
+                                    public void onClick(View v) {
+                                        Intent settingsIntent = new Intent();
+                                        settingsIntent.setAction(Settings.ACTION_APPLICATION_DETAILS_SETTINGS);
+                                        settingsIntent.setData(
+                                                Uri.fromParts(
+                                                        "package",
```
Comment (IanArb) [inline belvedere/src/main/java/zendesk/belvedere/BelvedereUi.java:230]: Just a small nit pick. Can we define a constant for this? It seems this class already defined a few constants already.

File: belvedere/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+    <string name="belvedere_dialog_gallery">Gallery</string>
+    <string name="belvedere_fam_desc_collapse_fam">Collapse Floating Action Menu</string>
+    <string name="belvedere_fam_desc_expand_fam">Expand Floating Action Menu</string>
+    <string name="belvedere_fam_desc_open_gallery">Import file from gallery</string>
+    <string name="belvedere_fam_desc_open_google_photos">Import image from Google Photos</string>
+    <string name="belvedere_image_stream_file_too_large">File too large</string>
+    <string name="belvedere_image_stream_title">Photo library</string>
+    <string name="belvedere_image_stream_unknown_app">Unknown</string>
+    <string name="belvedere_permissions_denied">To send attachments, you must grant permissions in your settings</string>
+    <string name="belvedere_navigate_to_settings" translatable="false">Settings</string>
```
Comment (ReguloSar) [inline belvedere/src/main/res/values/strings.xml:13]: Quick question: Does the build fail if we don't mark "Settings" as non-translatable.?

File: belvedere/src/main/java/zendesk/belvedere/BelvedereUi.java (hunk 1 above)
Comment (ahmedadeltito) [reply belvedere/src/main/java/zendesk/belvedere/BelvedereUi.java]: Nice catch 👌🏻

File: belvedere/src/main/res/values/strings.xml (hunk 2 above)
Comment (ahmedadeltito) [reply belvedere/src/main/res/values/strings.xml:13]: Just for now, we should translate it for all the supported languages we have in the library. I don't know how we can do that, yet that's why I labeled it as non-translatable for now until we translate it in another PR.

**Review by IanArb — APPROVED** [review]

**Review by ConorOD — APPROVED** [review]

**Comment by ConorOD** [comment]
🚢

**Review by AbdelrahmanTalat — APPROVED** [review]

**Pull request merged**